cmlreaders/cmlreader.py

```diff
@@ -140,10 +140,14 @@ def load_eeg(self, events: Optional[pd.DataFrame] = None,
         }
 
         if events is not None:
+            if rel_start is None or rel_stop is None:
+                raise ValueError("rel_start and rel_stop are required keyword"
```

Comment:
May be worth raising IncompatibleParametersError as defined in cmlreaders/exc.py instead